server.py

```diff
@@ -5,22 +5,24 @@
 import time
 from transformers import AutoModelForCausalLM, AutoTokenizer
 import torch
-from inference import generate
+from inference import Model, default_SYSTEM_MESSAGE
 import uvicorn
 import argparse
+import os
 
 app = FastAPI()
 
 
 class ChatInput(BaseModel):
     messages: List[Dict[str, Any]]
-    functions: Optional[List[Dict[str, Any]]]
+    functions: Optional[List[Dict[str, Any]]] = []
+    plugin_urls : Optional[List[str]] = [] ## cannot use openai client module for this param. coming soon.
```

Comment:
do you think we need custom client? Cause AFAIK, openai client library passes the args